src/features/unified-share-modal/utils/convertData.js

```diff
@@ -233,3 +233,27 @@ export const convertCollabsResponse = (
 
     return { collaborators };
 };
+
+/**
+ * Convert an enterprise users API response into an array of internal USM contacts.
+ *
+ * @param {UserCollection} contactsAPIData
+ * @param {string|null} currentUserID
+ */
+export const convertContactsResponse = (
+    contactsAPIData: UserCollection,
+    currentUserID: string | null,
+): Array<contactType> => {
+    const { entries } = contactsAPIData;
+
+    if (!entries || !entries.length) return [];
```

Comment:
you should be able to apply `.filter` to the return statement below so you avoid having multiple returns in this. More functional that way, and fewer branches to test.